tests/SampleValidation/Chat.cs

```diff
@@ -32,6 +32,9 @@ public async Task SuperbowlQuestions()
         string baseAddress = Environment.GetEnvironmentVariable("FUNC_BASE_ADDRESS") ?? "http://localhost:7071";
         string chatId = $"superbowl-{Guid.NewGuid():N}";
 
+        string functionKey = Environment.GetEnvironmentVariable("FUNC_KEY") ?? throw new InvalidOperationException("Missing environment variable 'FUNC_KEY'");
```

Comment:
IIRC, Functions has three types of keys, API keys, system keys, and a master key. Which type of key does this environment variable represent? Should we update the environment variable name to make this more clear?

Reply:
Its the default function key (also known as a host key), I have updated the variable name to FUNC_DEFAULT_KEY.
Let me know if it is good.